api/routes/userRouter.ts

```diff
@@ -0,0 +1,6 @@
+import { Router } from "express";
+import user from "../controllers/userController";
+
+export const router = Router();
```

Comment:
생각해보니까 어차피 index.ts에서 
router as userRouter로 하고 있는데
차라리  userRouter.ts에서 
```suggestion
const userRouter = Router();
export default userRouter; 
```
로 하는게 나을 수도 있을 것 같습니다.